src/tunnels/consumers.py

```diff
@@ -17,8 +17,10 @@
 from authorized_keys.models import ReverseServerAuthorizedKeys
 from authorized_keys.models import ReverseServerUsernames
 class TerminalConsumer(AsyncWebsocketConsumer):
-    child_pid = None
-    fd = None
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
I've quickly reviewed the original documentation, here's one question.
What's the point of calling super.init()? I didn't see this in the original example.
Is there just an ensurance that nothing go wrong?

Reply:
Yap. I haven't read the source code of Channel's AsyncWebsocket yet.
So here call the super method to avoid some unexpected traps.